README.md

```diff
@@ -4,8 +4,12 @@
 
 Are you new to the OpenAPI Specification?
 
-Read our [Getting started](https://learn.openapis.org/) page first.
+Read our [Getting Started](https://learn.openapis.org/) page first.
 
+<p>Original site customized based on a theme based on theme by Patrick Marsceill, distributed by an <a href="https://github.com/pmarsceill/just-the-docs/tree/master/LICENSE.txt">MIT license.</a></p>
```

Comment:
```suggestion
<p>Original site customized based on a theme by Patrick Marsceill, distributed by an <a href="https://github.com/pmarsceill/just-the-docs/tree/master/LICENSE.txt">MIT license.</a></p>
```

Reply:
I didn't write that sentence nor sourced the theme so can't say whether the indirection is accurate or not...

Reply:
let's open a separate issue for this part - it's a good point (thanks @ralfhandl !) but it's not related to the work in this pull request.